OpenEdX/Info.plist

```diff
@@ -29,5 +29,12 @@
 	<array/>
 	<key>UIViewControllerBasedStatusBarAppearance</key>
 	<true/>
+	<key>branch_universal_link_domains</key>
+	<array>
+		<string>edx.app.link</string>
```

Comment:
I guess we can because it's a codebase for openedX, and edX can be used as a placeholder. Although I'd make it configurable via the config file. Now the entry is being added from the config script.